docs/wallet/Desktop-Wallet-Guide.md

```diff
@@ -40,3 +40,68 @@ Due to the nature of UTXO, every time a transaction is made a couple of new Unsp
+- You have created a "hidden wallet" with the passphrase _letsUnlockAHiddenWallet_. You have transferred 900 ALPH to it.
+- You have created yet another "hidden wallet" with the passphrase _youCanNeverGuessThat!_. You have transferred the remaining 9'000 ALPH to it.
+
+An attacker threatens you to reveal the password to your _My Alephium Wallet_ wallet. You give it and the attacker steals the 100 ALPH in it. The attacker notices that the wallet offers the Passphrase feature and once again threatens you to reveal any additional Passphrases. At this point, you can deny that you use this feature and there will be no way for the attacker to prove whether you use it or not. Nevertheless, you decide to give them the passphrase _letsUnlockAHiddenWallet_ and the attacker steals the 900 ALPH. The attacker is happy that they stole all your funds and leaves you in peace. You have successfully managed to secure your safety and most of your funds' safety.
+
+### Best practices when using a Passphrase
+
+You must remember your Passphrase perfectly. Changing a single character (even, for example, from lower to upper-case), will result in the generation of a completely new wallet. As a result, it is imperative to:
+
+1. Choose a Passphrase that you can remember and enter it correctly before sending any funds to the generated wallet.
+2. Remember it character by character.
+
+[bip39-spec]: https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki#from-mnemonic-to-seed
+
+### Limitations of Passphrase-enabled wallets
+
+**You cannot (yet) use colored labels for your generated addresses. Any additionally generated addresses will need to be re-generated after every login.**
+
+The desktop wallet allows you to generate multiple addresses and select a colored label for each. This is helpful for organizing your funds into different "buckets". The metadata information about the addresses that you generated as well as their labels and their colors is stored in the local storage of the desktop wallet app.
+
+To guarantee the benefit of plausible deniability for our users, we cannot store metadata related to the Passphrase-enabled wallets, since we assume that an attacker has access to your wallet password, your 24-word secret phrase, as well as to the local storage of the app. If we did, the attacker could then prove whether you use the Passphrase feature or not, voiding plausible deniability. Thus, the Passphrase-enabled wallets cannot remember which addresses you have generated and what labels and colors you assign to each of them.
+
+In the future, we will be able to bring the metadata storage feature to the Passphrase-enabled wallets, once we implement the feature that will allow importing of Extended Public Keys. Then, you will be able to store additional addresses together with their labels and colors in the Passphrase-enabled wallets. Stay tuned and always keep your desktop wallet app up to date!
```

Comment:
I agree. if we really want plausible deniability we should not keep any metadata about the passphrase protected wallet. 

But I would not show the data of the wallet w/o passphrase on the one with passphrase in order not to confuse users. Just would say NO METADATA for passphrase wallet.

Reply:
This is not what we discussed in Neuchatel @polto. We agreed we'd go for no metadata because we don't yet implemented import of extended pub keys. Once we do that we can use the pub key as the key to the metadata. It was your idea :P

Reply:
Yes, all good, metadata is disabled at the moment when passphrase is used, as the wiki page explains.